### PR TITLE
Tidy debian packaging

### PR DESCRIPTION
**Pull request by lamby**

### CODE REVIEW

File: src/version.h [hunk 1]
```diff
@@ -1,5 +0,0 @@
-
-#ifndef RDM_VERSION
```
Comment (uglide) [inline src/version.h]: This is unrelated change, which breaks compilation of RDM. Please, remove this change. Thanks!

**Comment by lamby** [comment]
Not sure how version.h got deleted (!). Sorry about that.
